azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -340,26 +340,26 @@ public boolean shouldProxy() {
   private void registerCustomCredential(final Props props, final Credentials hadoopCred, final
   String userToProxy, final Logger jobLogger) {
     String credentialClassName = "unknown class";
-      try {
-        credentialClassName = props
-            .getString(Constants.ConfigurationKeys.CUSTOM_CREDENTIAL_NAME);
-        logger.info("custom credential class name: " + credentialClassName);
-        final Class credentialClass = Class.forName(credentialClassName);
-
-        // The credential class must have a constructor accepting 3 parameters, Credentials,
-        // Props, and Logger in order.
-        Constructor constructor = credentialClass.getConstructor (new Class[]
-            {Credentials.class, Props.class, Logger.class});
-        final CredentialProvider customCredential = (CredentialProvider) constructor
-              .newInstance(hadoopCred, props, jobLogger);
-        customCredential.register(userToProxy);
+    try {
```

Comment:
auto format change

Reply:
Consider making the format change in a separate PR first.